apis/proto/v1/payload/payload.proto

```diff
@@ -79,18 +79,22 @@ message Search {
     string request_id = 1;
     // Maximum number of result to be returned.
     uint32 num = 2 [ (validate.rules).uint32.gte = 1 ];
+    // Minimum number of result to be returned.
+    uint32 min_num = 3 [ (validate.rules).uint32.gte = 0 ];
     // Search radius.
-    float radius = 3;
+    float radius = 4;
     // Search coefficient.
-    float epsilon = 4;
+    float epsilon = 5;
+    // Search number of	agent ratio.
+    float ratio = 6;
     // Search timeout in nanoseconds.
-    int64 timeout = 5;
+    int64 timeout = 7;
+    // Disables asynchronous reranking
+    bool disable_async_rerank = 8;
     // Ingress filter configurations.
-    Filter.Config ingress_filters = 6;
+    Filter.Config ingress_filters = 11;
     // Egress filter configurations.
-    Filter.Config egress_filters = 7;
-    // Minimum number of result to be returned.
-    uint32 min_num = 8 [ (validate.rules).uint32.gte = 0 ];
+    Filter.Config egress_filters = 12;
```

Comment:
If I remember correctly, you told me that proto parameters should be added later, but what is the reason for rewriting this time?